source/extensions/filters/network/mysql_proxy/mysql_utils.h

```diff
@@ -51,6 +51,71 @@ class BufferHelper : public Logger::Loggable<Logger::Id::filter> {
   static DecodeStatus peekHdr(Buffer::Instance& buffer, uint32_t& len, uint8_t& seq);
 };
 
+/**
+ * MySQL auth method.
+ */
+enum class AuthMethod : uint8_t {
+  Unknown,
+  OldPassword,
+  NativePassword,
+  Sha256Password,
+  CacheSha2Password,
+  ClearPassword
+};
+
+// Helper function collection of old_password
+struct OldPassword {
+public:
```

Comment:
Methods and data are _public_ by default in _struct_.

Reply:
fixed.